submissions/Nik3264/Building-a-Tiny-JS-World/index.js

```diff
@@ -0,0 +1,79 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _https://github.com/Nik3264/a-tiny-JS-world_
+   Web app: _https://nik3264.github.io/a-tiny-JS-world/_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+   species: 'dog',
+   name: 'Toby',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'woof-woof!'
+ };
+
+ const cat = {
+   species: 'cat',
+   name: 'Vasya',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'mi-mi!'
+ };
+ const catwoman = {
+   species: 'catwoman',
+   name: 'Toby',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: cat.saying
+ };
+ const men = {
+   species: 'men',
+   name: 'Toby',
+   gender: 'male',
+   legs: 2,
+   hands: 2,
+   saying: 'hello!'
+ };
+ const woman = {
+   species: 'woman',
+   name: 'Tyna',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: 'wo hei!'
+ };
+
+ const arr = [dog, cat, catwoman, men, woman];
+
+ const str = arr.map(el=>{
+  el.name=`<strong>${el.name}</strong>`;
+   return Object.values(el).join(', ');
```

Comment:
There is a rule in the task: do not use `Object.values` and similar methods. They are not very reliable in terms of the keys order and presence. Find a way to set a list of keys by hand and work with it.

Reply:
I don't know how to do it yet... May I use  Object.keys() ? And then get to values throught  Object.keys() .... ?
Thank you so much.

Reply:
> and similar methods

This meant `Object.keys` too. And others like it.

The idea is almost right, you just have to figure out where to get an array of properties to show (hint: make it by yourself)

Reply:
Thank you! I have created an array of properties. now all properties are displayed.